src/init.js

```diff
@@ -14,11 +14,12 @@ module.exports = function () {
 
 function initialize() {
   setInterval(() => {
-    http.get(`http://dlockly.glitch.me/`);
+    http.get(`http://${process.env.PROJECT_DOMAIN}.glitch.me/`);
```

Comment:
Made it easier for me (or anyone) to add it. now it uses a custom glitch domain so I can easily run it and dont need to turn it back